spec/factories/form_objects/email_confirmation_form.rb

```diff
@@ -0,0 +1,12 @@
+FactoryBot.define do
+  factory :email_confirmation_form, class: "EmailConfirmationForm" do
+    send_confirmation { nil }
+    confirmation_email_address { nil }
+    notify_reference { Faker::Internet.uuid }
```

Comment:
```suggestion
    notify_reference {  SecureRandom.uuid }
```

Dont think we need faker for the uuid

Reply:
Done!